homeassistant/components/nextcloud/__init__.py

```diff
@@ -100,28 +100,31 @@ def setup(hass: HomeAssistant, config: ConfigType) -> bool:
         ncm = NextcloudMonitor(conf[CONF_URL], conf[CONF_USERNAME], conf[CONF_PASSWORD])
     except NextcloudMonitorError:
         _LOGGER.error("Nextcloud setup failed - Check configuration")
-
-    hass.data[DOMAIN] = get_data_points(ncm.data)
-    hass.data[DOMAIN]["instance"] = conf[CONF_URL]
-
-    def nextcloud_update(event_time):
-        """Update data from nextcloud api."""
-        try:
-            ncm.update()
-        except NextcloudMonitorError:
-            _LOGGER.error("Nextcloud update failed")
-            return False
-
+        result = False
```

Comment:
Just return False here. Then we can outdent below again.
```suggestion
        return False
```